<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepartmentWidgetClass</class>
 <widget class="QWidget" name="DepartmentWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DepartmentWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QLineEdit" name="descriptionLineEdit">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="ownDepartmentLabel">
         <property name="text">
          <string>Own Department</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="ownDepartmentListWidget"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="unassignedLabel">
         <property name="text">
          <string>Unassigned</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="unassignedListWidget"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="unassignedPushButton">
       <property name="text">
        <string>Top 3 Unassigned</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addUnPushButton">
       <property name="text">
        <string>Add Unassigned</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="nameLabel">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="nameLineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="emailLabel">
         <property name="text">
          <string>Email</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="emailLineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Interests</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="interestsLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="addPushButton">
       <property name="text">
        <string>Add Volunteer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
